perun-core/src/main/java/cz/metacentrum/perun/core/api/ConsentsManager.java

```diff
@@ -72,4 +75,18 @@
 	 * @throws PrivilegeException insufficient rights
 	 */
 	ConsentHub updateConsentHub(PerunSession perunSession, ConsentHub consentHub) throws ConsentHubNotExistsException, PrivilegeException, ConsentHubExistsException;
+	/**
+	 * Set consent status
+	 *
+	 * @param sess perun session
+	 * @param consentId id of consent
+	 * @param status status that should be set
+	 * @param userId id of user
+	 * @return consent
+	 * @throws ConsentNotExistsException
+	 * @throws PrivilegeException
+	 * @throws UserNotExistsException
+	 * @throws InvalidConsentStatus
+	 */
+	Consent changeConsentStatus(PerunSession sess, int consentId, ConsentStatus status, int userId) throws ConsentNotExistsException, PrivilegeException, UserNotExistsException, InvalidConsentStatus;
```

Comment:
Do we need to pass userId? Can't we always get the relevant User object from the Consent object?